src/Hydro/CMakeLists.txt

```diff
@@ -51,4 +51,8 @@ list( APPEND locallibs hydro)
 # 2. hydro is added last except for core, so that all the modules that were used got here.
 message(STATUS "Mod libraries: ${schismmodlibs}")
 add_dependencies(hydro core ${schismmodlibs})
-target_link_libraries(hydro core ${schismmodlibs})
+target_link_libraries(hydro core ${schismmodlibs} ${NetCDFLIBS} MPI::MPI_Fortran)
```

Comment:
To provide some rationale for the change in this file, I'm adding the dependencies here because the Hydro library actually carries these dependencies, and specifying them here makes them transitive to anything that depends on it.